fringe-helper.el

```diff
@@ -181,8 +184,8 @@ input automatically."
 (defun fringe-helper-modification-func (ov after-p beg end &optional len)
   ;; Sometimes this hook is called with a deleted overlay.
   (when (overlay-start ov)
-    (setq beg (max beg (overlay-start ov)))
-    (setq end (min end (overlay-end ov)))
+    (setq beg (max (min beg end) (overlay-start ov)))
+    (setq end (min (max beg end) (overlay-end ov)))
```

Comment:
D'oh! Yes, you're quite right.

On Thursday, April 30, 2015, Nikolaj Schumacher <notifications@github.com>
wrote:

> In fringe-helper.el
> <https://github.com/nschum/fringe-helper.el/pull/4#discussion_r29477355>:
>
> > @@ -181,8 +184,8 @@ input automatically."
> >  (defun fringe-helper-modification-func (ov after-p beg end &optional len)
> >    ;; Sometimes this hook is called with a deleted overlay.
> >    (when (overlay-start ov)
> > -    (setq beg (max beg (overlay-start ov)))
> > -    (setq end (min end (overlay-end ov)))
> > +    (setq beg (max (min beg end) (overlay-start ov)))
> > +    (setq end (min (max beg end) (overlay-end ov)))
>
> If end < beg, then both beg and end will be equal to the original end
> after line 187. The original beg will already be lost in line 188.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nschum/fringe-helper.el/pull/4/files#r29477355>.
>